opm/input/eclipse/EclipseState/Util/OrderedMap.hpp

```diff
@@ -103,7 +153,17 @@ class OrderedMap {
     T& get(const K& key) {
         auto iter = m_map.find( key );
         if (iter == m_map.end())
-            throw std::invalid_argument("Key not found:");
+        {
+            using namespace std::string_literals;
+            auto startsWithSame = OrderedMapDetail::findSimilarStrings(key, m_vector);
+            if (!startsWithSame.empty())
+            {
+                startsWithSame = " Similar entries are "s +
+                    startsWithSame + "."s;
```

Comment:
It's not a huge deal but for what it's worth creating these temporary objects of type `std::string` does cost (a tiny bit) more than just forming the final directly string through the expression
```C++
startsWithSame = " Similar entries are " + startsWithSame + '.';
```
given that `std::basic_string<>` has overloads of `operator+` that accept mixes of `CharT*` and `CharT`.